triangle.json

```diff
@@ -0,0 +1,98 @@
+{
+    "#": [
+        "Pursuant to discussion in #202,",
+        "we have decided NOT to test triangles where a + b = c.",
+        "e.g: (2, 4, 2, Isosceles), (1, 3, 4, Scalene).",
```

Comment:
One possible course of action may be to clarify "we have decided NOT to test triangles where a, b, c are all positive and a + b = c" because as I see it, the point of (0, 0, 0, Illegal) and (10, 10, 0, Illegal) are to test that 0 is not a valid side length, not just to be a case like (2, 4, 2, Isosceles) or (1, 3, 4, Scalene).

However, it may be the case that cases involving zero should also get thrown out for the same reasons as in #202: they raise questions about degeneracy. Can we call it a triangle even if some side lengths are zero? And unrelated to this exercise but related to the previous question: Can we consider every triangle to be a quadrilateral with exactly one side length zero? I don't have a definitive answer.

- If we can find a definitive reason to reject 0 as a valid side length, (0, 0, 0, Illegal) and (10, 10, 0, Illegal) should stay and the above sentence should be amended.
- If we cannot, the cases should go.